hoppscotch/hoppscotch
# HBE-326 feature: server configuration through GraphQL API

Closes HBE-326

### Description

This PR introduces the ability (in backend service) to update server configurations (`GOOGLE_CLIENT_ID`, `MAILER_SMTP_URL`, `VITE_ALLOWED_AUTH_PROVIDERS`, etc) from the UI (Admin Dashboard).

Graphql Mutation and Query lists: https://linear.app/hopp/issue/HBE-326/shc-hands-on-implementation-server-configuration

Details of backend implementation: https://www.notion.so/hoppscotch/Env-Variable-Configuration-from-UI-609016e231fc4f8ab85d3b2d83926ec1




### Checks


### Testing Instruction

1. You can test/run the above query and mutation. 
2. Check the InfraCondig table (in DB) which should reflect the new value.
3. Check after any mutation, the backend server will restart

### Additional Information

Nil
